pkg/operator/resources/validations.go

```diff
@@ -96,7 +96,7 @@ func ValidateClusterAPIs(apis []userconfig.API, projectFiles spec.ProjectFiles)
 				return errors.Wrap(err, api.Identify())
 			}
 			if err := validateK8s(api, virtualServices, maxMem); err != nil {
-				return errors.Wrap(err, api.Identify())
+				return err
```

Comment:
Since the api context is being created in this function and is being passed in, would it make sense to wrap the error with `api.Identify()` here instead of below?

Reply:
I was running on the impression that `validateEndpointCollisions` had the API identity placed as the 3rd argument for `errors.Wrap` function, thus thinking that it's not possible to move it up the call stack, LOL. Moved it up.